modules/ROOT/pages/batch-processing-concept.adoc

```diff
@@ -186,6 +190,31 @@ Every processed record of the Batch Job Instance starts with the same variables
 In the On Complete phase, none of these variables (not event the original ones) are visible. Only the final result is available in this phase. Moreover, since the Batch Job Instance executes asynchronously from the rest of the flow, no variable set in either a Batch Step or the On Complete phase will be visible outside the Batch Scope.
 
 
+=== Scheduling Strategy
+
+The Batch Job scope provides Scheduling Strategy options for controlling the
+execution of job instances.
+
+* ORDERED_SQUENTIAL (the default): If several job instances are in an executable
+state at the same time, the instances execute one at a time based on their
+creation timestamp.
++
+Note that there is no guarantee that the records will be executed in order.
+This setting does not control the execution order of records in the batch job,
+nor does it take into account the number of records each instance contains.
++
+* ROUND_ROBIN: This setting attempts to execute all available instances of a
+batch job using a round-robin algorithm to assign the available resources. It
+works for the use case in which jobs are triggered manually and you want them to
+run concurrently, such as one-way synchronization. This setting was introduced
+before the current default, ORDERED_SQUENTIAL.
+
+A Mule application can contain more than one batch job configuration. These
+separate batch jobs can run at the same time because the scope of each
```

Comment:
This entire paragraph is incorrect, what's said here is not true. It should just say that each app can contain several batch job definitions and that each one of them has its own scheduling strategy.

Reply:
done